cli/server/getNarrative.js

```diff
@@ -3,6 +3,7 @@ const parseNarrative = require('./parseNarrative').default;
 const path = require("path");
 const fs = require("fs");
 const utils = require("../utils");
```

Comment:
I don't think it will help since it still call the `console.log` and `console.error` inside the utils
As I understand, the security tool recommends we use special logging tools that eliminate the break line character